src/clj/rems/poller/email.clj

```diff
@@ -261,8 +263,8 @@
         email (assoc email-spec
                      :from (:mail-from env)
                      :to (or (:to email-spec)
-                             (util/get-user-mail
```

Comment:
nit: This could be cleaned up a bit further by deleting util/get-user-mail (as well as util/get-username) as neither is used after this patch. It also unifies things a bit since both functions use the "old-style" :eppn/:commonName access.

Reply:
good catch!

Reply:
done